install.sh

```diff
@@ -230,13 +227,6 @@ __install_mongodb() {
 }
 
 __install_rita() {
-	# Set _RITA_VERSION if it's not set
```

Comment:
We no longer want to get the latest version from Github as the version is hard coded.